drivers/applications/tla2528_output_pin_demo.cpp

```diff
@@ -0,0 +1,51 @@
+#include "../hardware_map.hpp"
+#include <array>
+#include <libhal-util/bit.hpp>
+#include <libhal-util/i2c.hpp>
+#include <libhal-util/serial.hpp>
+#include <libhal-util/steady_clock.hpp>
+#include <libhal/output_pin.hpp>
+#include <libhal/units.hpp>
+#include <tla2528.hpp>
+#include <tla2528_adapters.hpp>
+
+using namespace hal::literals;
+using namespace std::chrono_literals;
+
+namespace sjsu::drivers {
+
+void application(application_framework& p_framework)
+{
+
+  bool demo_open_drain = false;
```

Comment:
```suggestion
  constexpr bool demo_open_drain = false;
```

Might as well make this constexpr if you aren't going to change this variable.

Reply:
Should I also make the hal::input_pin::settings and hal::output_pin::settings constexpr too?

Reply:
Yes! That is also a good idea as well.